front/next.config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
           {
             key: "Content-Security-Policy",
             value:
-              "frame-ancestors 'self' https://*.salesforce.com https://*.force.com;",
+              "frame-ancestors 'self' https://*.salesforce.com https://*.force.com chrome-extension://ijppfkjmlibdkjpiakcjbhdcofbamoni;",
```

Comment:
I'm going to do a PR to force the id on the manifest so we have the same one 😬

Reply:
actually not needed anymore